pkg/planner/core/operator/physicalop/task_base.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2025 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package physicalop
+
+import (
+	"math"
+
+	"github.com/pingcap/tidb/pkg/util/context"
+)
+
+// SimpleWarnings is a simple implementation of Warnings interface.
+type SimpleWarnings struct {
```

Comment:
What's the benefits to move it from package core to package physicalop?

Reply:
+1 i think moving task_base.go file can be done in a whole.